spec/factories/timeline_entries.rb

```diff
@@ -0,0 +1,12 @@
+FactoryGirl.define do
+  factory :timeline_entry do
```

Comment:
Do we need a factory for this? These should never be instantiated directly, right? More of an abstract class...